paired-fastq-to-unmapped-bam-fc.wdl

```diff
@@ -0,0 +1,145 @@
+## Copyright Broad Institute, 2018
+## 
+## This WDL converts paired FASTQ to uBAM and adds read group information 
+##
+## Requirements/expectations :
+## - Pair-end sequencing data in FASTQ format (one file per orientation)
+## - One or more read groups, one per pair of FASTQ files  
+## - A readgroup.list file with the following format :  
+##   ``readgroup   fastq_pair1   fastq_pair2   sample_name   library_name   platform_unit   run_date   platform_name   sequecing_center``
+##
+## Outputs :
+## - Set of unmapped BAMs, one per read group
+## - File of a list of the generated unmapped BAMs
+##
+## Cromwell version support 
+## - Successfully tested on v32
+## - Does not work on versions < v23 due to output syntax
+##
+## Runtime parameters are optimized for Broad's Google Cloud Platform implementation. 
+## For program versions, see docker containers. 
+##
+## LICENSING : 
+## This script is released under the WDL source code license (BSD-3) (see LICENSE in 
+## https://github.com/broadinstitute/wdl). Note however that the programs it calls may 
+## be subject to different licenses. Users are responsible for checking that they are
+## authorized to run all programs before running this script. Please see the docker 
+## page at https://hub.docker.com/r/broadinstitute/genomes-in-the-cloud/ for detailed
+## licensing information pertaining to the included programs.
+
+# WORKFLOW DEFINITION
+workflow ConvertPairedFastQsToUnmappedBamWf {
+  File readgroup_list  
+  Array[Array[String]] readgroup_array = read_tsv(readgroup_list) 
+  String ubam_list_name = basename(readgroup_list,".list") + "unmapped.bam.list"
+  
+  String? gatk_docker_override
+  String gatk_docker = select_first([gatk_docker_override, "broadinstitute/gatk:latest"])
```

Comment:
can the default docker be the ones in gcr instead of dockerhub? Especially as this is the FireCloud version?

Reply:
i've modified the ones in FC to use gcr. The idea was to have the ones in git use docker to avoid any authentication issues for users running on different platforms.

Reply:
oops no i didn't , I forgot this was the FC version. thanks for catching this.